docs/index.rst

```diff
@@ -46,6 +47,23 @@ Data pattern is as follows:
 | Elasticsearch sink connector        | ``WIKIPEDIABOT`` (from ksqlDB) | Elasticsearch/Kibana                  |
 +-------------------------------------+--------------------------------+---------------------------------------+
 
+How to use this tutorial
+------------------------
+
+We suggest going following the ``cp-demo`` tutorial in order:
```

Comment:
I'm not sure of the intent of "going following", either "going through" or just "following"?

Reply:
Good catch, now resolved.